### PR TITLE
PR for React capstone project - Metrics webapp

### PR DESCRIPTION
**Pull request by jose-Abel**
Hello, 
This is my PR for the React capstone project - Metrics webapp Project, where I've done the following:
- A single page application (SPA) built with React and Redux.
- The data retrieved from the API is stored in the Redux store.
- I created a filter component, InputFilter, that filter the data retrieved from the API based on the search parameter.
- Every page (the main page and the pages for each item) have a unique route within the SPA.
- The project is be deployeded and accessible online.
- I followed the design guidelines of https://www.behance.net/sakwadesignstudio.
- I created unit tests for pure functions in your Redux code.
- I created integration tests for my application using the React Testing Library.
Cheers.

### CODE REVIEW

File: src/components/layout/home/HomePage.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getCryptosList } from '../../../redux/cryptos/cryptos';
+import Crypto from '../../crypto/Crypto';
+import InputFilter from '../../filter/InputFilter';
+import Header from '../header/Header';
+import classes from './HomePage.module.css';
+
+const HomePage = () => {
```
Comment (rloterh) [inline src/components/layout/home/HomePage.js:9]: - [x] The styling of the list on your Homepage does not follow the provided [design template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), and therefore difficult to differentiate the details of each list item. Kindly note we as much as possible want your design to match the provided [template](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)). Therefore kindly ensure that cells of your list follow the **alternating dual-color pattern**. Each cell of the list item should have surrounding cells having **different (alternating) background color(s)**.


**Your design:**
image



**The original design:**
image

File: src/components/filter/InputFilter.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import classes from './InputFilter.module.css';
+
+const InputFilter = ({ setSearched }) => {
```
Comment (rloterh) [inline src/components/filter/InputFilter.js:5]: Good job for implementing the search filter, however, it needs further improvement.
- [x] When part of a search term is keyed in the search field, the filter result does not display all possible matches. For instance, when I enter `eth` in the search field, the only displayed result is `Tether` whereas other coins such as `Ethereum` and `Lido Staked Ether` also have the term `eth` in their names. Kindly ensure to improve upon this aspect.

**Searching with the term `eth`:**
image

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Abel, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to complete this project but you are almost there! 🚦  🚦  

## Good Points 👍 
- Descriptive Pull Request
- App is correctly deployed
- Good video presentation of the project
- Redux is effectively used to manage state data
- Test cases are passing
- Good ReadMe

## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 4 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 3]
```diff
@@ -1 +1,112 @@
...
+
+```
+    git clone 'repository-link'
+```
+
+That's all, you are ready to go!
+
+# Live Demo
+
+[Metrics Web App](https://jose-abel-metrics-webapp.netlify.app/)
```
Comment (TedLivist) [inline README.md:53]: - [x] Good job implementing the changes requested. Locally, they work fine now. However, looking at the image below, your changes are not pushed to your live demo yet. Kindly push them.

jose-livedemo

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @jose-Abel,

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 3 above)
Comment (rloterh) [reply README.md:53]: These issues are well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Abel, 

Your project is complete! Great job for making the necessary changes recommended in the previous review. There is nothing else to say other than... it's time to merge it 🍾 :shipit:.
Congratulations! 🎉🥂

<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 3 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**